server/remote/msgraph/subscription.go

```diff
@@ -14,11 +16,16 @@ import (
 const subscribeTTL = 10 * time.Minute
 
 func (c *client) CreateMySubscription(notificationURL string) (*remote.Subscription, error) {
+	b := make([]byte, 96)
+	rand.Read(b)
+	clientState := base64.URLEncoding.EncodeToString(b)
```

Comment:
1/5 put this logic in its own function `newRandomString`

Reply:
Should it be in the same file?

Reply:
Sounds good to me :+1: